scripts/data_prep/convert_finetuning_dataset.py

```diff
@@ -170,12 +188,22 @@ def main(args: Namespace) -> None:
                 'include the "--skip-preprocessing" flag to avoid this error.'
             )
 
-    columns = ['prompt', 'response']
+    tokenizer = None
+    args.tokenizer_kwargs.update({'model_max_length': args.max_seq_len})
```

Comment:
lets pull the `tokenizer_kwargs` out of args and define in a variable before updating, so that the `args` remains unmodified.